packages/web-components/src/components/content-group-horizontal/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,70 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+
+# `<dds-content-group-horizontal>`
+
+> The Content Group - Horizontal block pattern contains a collection of
+> `<dds-content-item-horizontal>` content items.
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/components/content-group-horizontal)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/components/content-group-horizontal)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/content-group-horizontal/content-group-horizontal';
+import '@carbon/ibmdotcom-web-components/es/components/content-block/content-block-heading';
+import '@carbon/ibmdotcom-web-components/es/components/content-item-horizontal/content-item-horizontal';
+```
+
+### HTML
+
+```html
+   <dds-content-group-horizontal>
+      <dds-content-block-heading>Curabitur malesuada varius mi eu posuere</dds-content-block-heading>
+      <dds-content-item-horizontal .copy='Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin.'>
+        <slot slot="eyebrow">Lorem ipsum</slot>
```

Comment:
`<slot>` is for defining a slot in shadow DOM, instead of using it. Please refer to: #4406

```suggestion
        <span slot="eyebrow">Lorem ipsum</span>
```